clients/src/main/java/org/apache/kafka/common/metrics/Metrics.java

```diff
@@ -76,6 +76,10 @@ public class Metrics implements Closeable {
     private final Time time;
     private final ScheduledThreadPoolExecutor metricsScheduler;
     private static final Logger log = LoggerFactory.getLogger(Metrics.class);
+    // Allowing the initial delay and period of the metrics scheduler to be changed
+    // so that metrics can be expired faster in tests
+    private static long metricsSchedulerInitialDelay = 30;
+    private static long metricsSchedulerPeriod = 30;
```

Comment:
nit:  append "Seconds" to names?  Many similar constants are in milliseconds.

Reply:
Fixed. Thanks for the suggestion.